### DISCUSSION TITLE
Migration from v0.44.0 Entity Node to latest version

### DISCUSSION THREAD

**Discussion by Artekus** [zachowj/node-red-contrib-home-assistant-websocket#717] [Q&A]
Is there any recommended way to migrate from the "Entity node is being deprecated" to the individual entity nodes? The only way I see to do it is one by one but I have a ton of them and once you update you can't see the old entity node configuration details to be able to just copy and paste to the new individual entity node. 

Recommendations?

**Comment by zachowj** [comment]
There should be a button at the top of NR that convert all the old nodes at once.
